label_studio_ml/examples/segment_anything_model/README.md

```diff
@@ -14,6 +14,21 @@ To start the server with lightweight mobile version of SAM, run the following co
 docker-compose up
 ```
 
+### GPU Support
+
+By default, the docker-compose file runs the model on the CPU. If you have a GPU, you can enable it by adding the following lines in `docker-compose.yml`:
+
+```yaml
+    environment:
+      - NVIDIA_VISIBLE_DEVICES=all
```

Comment:
Is this line suggested for other yaml files? it doesnt appear in the `segment_anything_model` example.

Reply:
added to the docker-compose as well